plugins.json

```diff
@@ -3785,6 +3785,68 @@
     },
     "updated_at": "2019-06-12T19:04:44Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-todo",
+    "icon_data": "",
+    "download_url": "https://github.com/mattermost/mattermost-plugin-todo/releases/download/v0.2.0/com.mattermost.plugin-todo-0.2.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-todo/releases/tag/v0.2.0",
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl6fK3MACgkQ0bVLR6XO/sR2JQ/8CUdVcqBLXStJE8g+VbHUUT0hfBnijIPzbiPVzHjnkWIz6kY6nUrfykg6nSNN5rhverw0ovOLEwrusrKKb0997xvn60Ekmsyfrr3b2V+ZCVvgdG678lJWXnhqu6n/xtCW6GCTTVZB42Jnr4Ri5VVwNL0mSE6Rc4urlWnMApasKVi4tl3trbvVyOJrKUwtlgNXHYnRkZOVcmc64AxG7DBhXawmuNTUbZ6f9WaC8/F9ZxxErWxERN0rlH/rSdYsRG/CIIYxCJ63R9YT2xRMcJkiMBt2bh4zXKazHfw5/ozadyG7TEQVnPrNS+xZvj4JkgKLw5c3NwcIORONujnL8JINlOKUJf+xmBdNPGTwS2WWeDp6rSx0xPmSttXi8b+glBXvq80lrzSz1CLvSr/MOwB3yeuMl1EAKhPvwrUm49uGt2dsIgaHaImbAXXzs277OjuG4ra6bNKPpgjh7mjr5opnDRai3r1cqOMKPQnevBXxQCcRPbIUfxr3tVKOQSaG2cTFgot57yebnUGwYNFRD+D6oAkgTh2nIm95QPqLsQ3CpawPiR9pz9VIX0RVoQg0+OYKAGql8GjXH97cj4EZZnlDLtBIxol0mmGVxndNFYhYesnQxF9l2LroG/8jown+gqQorOcS5cv4RSGVl31QftWNgiDALr7mH2ionJFFn0vVjiA=",
+    "manifest": {
+      "id": "com.mattermost.plugin-todo",
+      "name": "Todo Plugin",
+      "description": "This plugin makes it easy to keep track of Todo issues and get daily reminders.",
+      "version": "0.2.0",
```

Comment:
I'm wondering if we should make clear that this plugin is still in beta, breaking changes might happen and the plugin is not production ready. See also https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/#beta-plugins


Please note that some code changes would be needed to add a "Beta" label. I'm happy to work on them if we decide to go this path.

Reply:
Moving the discussion to https://community-release.mattermost.com/core/pl/mgrez65gjfngbbjqqho9ka1xte